bin/chef-zero

```diff
@@ -55,6 +55,11 @@ OptionParser.new do |opts|
     options[:log_file] = value
   end
 
+  opts.on("--enterprise", "Whether to run in enterprise mode") do |value|
```

Comment:
I realize this PR is just the chef-zero part, but will we need to push this config option up to `chef-client -z`?